docs/source/project_docs/network.rst

```diff
@@ -216,6 +216,34 @@ Please also note that the network consistency triggers will NOT work before
 spatial indices have been created and/or if the editing is being done on a
 platform that does not support both RTree and Spatialite.
 
+.. _importing_from_gmns:
+
+Importing from files in GMNS format
+-----------------------------------
+
+This feature is still under development.
```

Comment:
Isso dá a imrpessão que estamos no meio do desenvolvimento, o que não será verdade quando essa versão for publicada. O que você pode falar é que não é "fully featured", mas só no caso de estar faltando algumas coisas de fato

Reply:
Alterado. Indiquei o que se tem desde o mês da implementação ("As of July 2022, ...").